audeer/core/utils.py

```diff
@@ -1,3 +1,4 @@
+from distutils.version import LooseVersion
```

Comment:
If we want to keep imports sorted, this should go after `import copy`

Reply:
Good p[oint, changed it.